src/cplscheme/CompositionalCouplingScheme.hpp

```diff
@@ -126,6 +124,12 @@ class CompositionalCouplingScheme final : public CouplingScheme {
    */
   bool hasDataBeenReceived() const final override;
 
+  /// @copydoc CouplingScheme::storeReceiveData
```

Comment:
This is done automatically for virtual functions.

Reply:
Done. abdd415a50a8edb822189ae63462849e73dea404